src/runtime_src/xdp/profile/plugin/aie_trace/aie_trace_plugin.cpp

```diff
@@ -484,88 +499,91 @@ namespace xdp {
       // 1. Reserve and start core module counters
       //
       int numCoreCounters = 0;
-      XAie_ModuleType mod = XAIE_CORE_MOD;
-      for (int i=0; i < coreCounterStartEvents.size(); ++i) {
-        auto perfCounter = core.perfCounter();
-        auto ret = perfCounter->initialize(mod, coreCounterStartEvents.at(i),
-                                           mod, coreCounterEndEvents.at(i));
-        if (ret != XAIE_OK) break;
-        ret = perfCounter->reserve();
-        if (ret != XAIE_OK) break;
+      if (useCounters) {
+        XAie_ModuleType mod = XAIE_CORE_MOD;
+        for (int i=0; i < coreCounterStartEvents.size(); ++i) {
+          auto perfCounter = core.perfCounter();
+          auto ret = perfCounter->initialize(mod, coreCounterStartEvents.at(i),
+                                            mod, coreCounterEndEvents.at(i));
+          if (ret != XAIE_OK) break;
```

Comment:
I know that this code has the body of the if statement on a common line.   And it is used in this file, going forward we are working on having the XRT code follow the google coding style, which has it on its own separate line.

Reply:
added line break